Added README, .editorconfig; updated issues links

cxw42 opened the pull request:
- Added a README.md converted from script/perlver, for the sake of
  the GitHub repo.
- Added the GitHub issues link alongside the the CPAN issues link
- Added a .editorconfig file to help with consistency in formatting.
  As far as I can tell from the source, it's Unix linefeeds, tab
  indents, and four-space tabs.  I got the four-space tabs from
  [lib/Perl/MinimumVersion.pm:459--460](https://github.com/cxw42/Perl-MinimumVersion/blob/341cf795aba51fad79b4126fd9903f774024522b/lib/Perl/MinimumVersion.pm#L459), which don't indent correctly
  with two-space tabs.  Of course, I could be wrong!

zmughal commented:
The `README.md` is no longer needed as there is a generated `README.pod` via #23. The rest of the changes look fine to me, but will need @neilb's input.

cxw42 commented:
@zmughal Thanks for the update!  I have modified the PR.
- I no longer create a README
- I discovered MinimumVersion.pm had mixed leading indents, so updated it to use all tabs.
- I updated the copyright dates and made sure they were in sync between MinimumVersion.pm and perlver.

`dzil regenerate` and `dzil test` pass.
